index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const server = express();
+const logger = require('morgan');
+const helmet = require('helmet');
+const PORT = 4000;
+const dbhelper = require('./dbhelper')
+const bcrypt = require('bcryptjs');
+server.use(express.json());
+server.use(helmet());
+server.use(logger('dev'))
+server.disable("etag");
+
+server.post('/api/register', (req, res) => {
+    const newUser = req.body;
+    console.log(newUser)
+    if (!newUser.password || !newUser.username) {
+        return res
+            .status(404)
+            .json('Please provide Username and/or password')
+    }
+    newUser.password = bcrypt.hashSync(newUser.password, 14);
```

Comment:
I would make this into another variable. Overwriting the variable stops it from being used and it might be helpful to have this variable remain pure during this process.